pandas/tests/resample/test_base.py

```diff
@@ -274,3 +274,72 @@ def test_resample_quantile(series):
     result = s.resample(freq).quantile(q)
     expected = s.resample(freq).agg(lambda x: x.quantile(q)).rename(s.name)
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "_index_factory,_series_name,_index_start,_index_end", [DATE_RANGE, PERIOD_RANGE]
+)
+@pytest.mark.parametrize(
```

Comment:
@ganevgv rather than nesting a dict, just make data, index paramters e.g.

```
@pytest.mark.parameterize('freq, data, index')
```

otherwise this is very hard to follow.

Reply:
@jreback - removed the nested dict and changed the format to: 
```
@pytest.mark.parameterize("freq, result_name, result_data, result_index, result_freq")
```